3.Tests/UI/Hooks/Hooks.cs

```diff
@@ -36,17 +47,40 @@ public void SessionDisposal()
         GenericWebDriver.Dispose();
     }
 
-    [BeforeScenario("create.project")]
+    [BeforeScenario("create.project", Order = 1)]
     public void CreateProject()
     {
         string payload = $"{{ \"Content\": \"{_projectName}\" }}";
 
         _client.AddAuthenticator(ConfigModel.TODO_LY_EMAIL, ConfigModel.TODO_LY_PASS);
 
-        RestResponse response = _client.DoRequest(Method.Post, _url, payload);
+        RestResponse response = _client.DoRequest(Method.Post, _urlProject, payload);
         ProjectModel? projectModel = JsonSerializer.Deserialize<ProjectModel>(response.Content!);
 
         _scenarioContext[ConfigModel.CurrentProject] = _projectName;
         _scenarioContext[ConfigModel.CurrentProjectPayload] = projectModel;
     }
+
+    [BeforeScenario("create.item", Order = 2)]
+    public void CreateAnItem()
+    {
+        string payload = $"{{ \"Content\": \"{_itemName}\", \"ProjectId\": {_projectId} }}";
+        _client.AddAuthenticator(ConfigModel.TODO_LY_EMAIL, ConfigModel.TODO_LY_PASS);
+
+        RestResponse response = _client.DoRequest(Method.Post, _urlItem, payload);
+        if (!response.IsSuccessful)
+        {
+            throw new Exception("Error: Bad Request");
```

Comment:
In order to differentiate request error (in this case an error on creating an item) you should be more specific on the exception message. Something like "Error: create item request failed", you might think of something better